doc/man/task-sync.5.in

```diff
@@ -30,7 +30,7 @@ the existing replica, and run `task sync`.
 
 .SS When to Synchronize
 
-Taskwarrior can perform a sync operation at every garbage collection (gc) run.
+Taskwarrior can perform a sync operation at every rebuild operation.
 This is the default, and is appropriate for local synchronization.
```

Comment:
Hm, this isn't actually true! I don't remember ever implementing this, so I don't know why I wrote it.
https://github.com/GothenburgBitFactory/taskwarrior/issues/3534 (so, no change required in this PR)

Reply:
Got it. This change was removed from the PR.